src/containers/connected-pipeline/connected-request-location/connected-request-location.js

```diff
@@ -1,28 +1,20 @@
 import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import * as PipelineActions from '../../../actions/pipeline-actions';
```

Comment:
We could probably reduce our import statements with this.

https://stackoverflow.com/questions/45213279/how-to-avoid-using-relative-path-imports-redux-action-action1-in-cre

Reply:
Can you create a card for that? We could do that in a new branch

Reply:
done